TwitchPlaysAssembly/Src/UI/TwitchBomb.cs

```diff
@@ -694,8 +697,17 @@ private void HandleStrikeChanges()
 		}
 	}
 
+	private IEnumerator FlipBombCooldown() //Ideally this would be done with a System.Timers.Timer, but that's explicitly multithreaded, and unity doesn't play well with it
+	{
+		yield return new WaitForSeconds(TwitchPlaySettings.data.BombFlipCooldown);
+		_flipEnabled = true;
+	}
+
 	public IEnumerator TurnBomb()
 	{
+		if (TwitchPlaySettings.data.BombFlipCooldownEnabled && !_flipEnabled)
```

Comment:
Instead of having a separate enabled setting, we could allow users to set the cooldown to zero:
```suggestion
		if (TwitchPlaySettings.data.BombFlipCooldown > float.Epsilon && !_flipEnabled)
```